challenges/4-loopcardio/loopcardio.js

```diff
@@ -13,7 +15,9 @@ Precondition: n > 0, and n will be an integer.
 */
 
 function printEvenNumbersUntil(n) {
-    // implement this
+    for (let i = 1; i % 2 == 0 && i <= n; i++) {
+        console.log(i);
```

Comment:
ah this is incorrect. Remember that the loop will break and end when the middle conditional returns false.

Reply:
Originally you asked if I can do this without the if statement - 
I ended up using `condition ? return if true : return if false`. Verified on RunJS this time!

Were you pushing me to use an alternative to if statements, use the while loop, or use arithmetic?

Reply:
Okay here's a better hint that'll probably give it away: instead of `i++`, what else could you put there?